app/assets/javascripts/discourse/app/widgets/header.js

```diff
@@ -85,18 +85,29 @@ createWidget("header-notifications", {
         !user.get("read_first_notification") &&
         !user.get("enforcedSecondFactor")
       ) {
-        contents.push(h("span.ring"));
         if (!attrs.active && attrs.ringBackdrop) {
+          contents.push(h("span.ring"));
           contents.push(h("span.ring-backdrop-spotlight"));
           contents.push(
             h(
               "span.ring-backdrop",
               {},
-              h(
-                "h1.ring-first-notification",
-                {},
-                I18n.t("user.first_notification")
-              )
+              h("h1.ring-first-notification", {}, [
+                h("span", {}, I18n.t("user.first_notification")),
+                h("br"),
```

Comment:
I am not a big fan of using `<br>` for formatting. Could we use css for this instead?